.github/workflows/dev-release.yml

```
@@ -100,23 +116,22 @@ jobs:
+          USE_LOCAL_EMBEDDINGS: 'true'
+          TTS_ENDPOINT: 'https://inference.tinfoil.sh/v1/audio/speech'
+          BUILD_CHANNEL: 'dev'
+          VITE_DISABLE_ONBOARDING: 'false'
+          VITE_DISABLE_HOLONS: 'false'
+          VITE_DISABLE_TASKS: 'false'
+          VITE_DISABLE_CONNECTORS: 'false'
+          VITE_DISABLE_VOICE: 'false'
         run: make build-dev-mac-silicon
 
       - name: Create dev release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)